includes/class-newspack-popups-model.php

```diff
@@ -745,13 +728,10 @@ public static function get_access_attrs( $popup ) {
 	 * @param object $popup Popup.
 	 */
 	private static function render_permanent_dismissal_form( $element_id, $popup ) {
-		$dismiss_text = self::get_dismiss_text( $popup );
-		if ( ! $dismiss_text ) {
-			return '';
-		}
```

Comment:
Still, this should return an empty string if the dismiss text is empty - otherwise a form element will be rendered with no content.

Reply:
Ah, good point. Re-added in 08947e8.